### DISCUSSION TITLE
Correction flag

### DISCUSSION THREAD

**Discussion by deez2009** [algertc/ALPR-Database#44] [Ideas]
ALPR aren't always accurate, in fact, I think we all struggle with improving this.  I think one way we can improve this is to export the images and labels to train a more localized model for OCR, and to do that, we have to know if the plates in the DB are correct or not.  One way  that can we done is to flag plates that have been edited.   We can also use this flag as stats to help tune cameras for LPR--number of plates read vs number of plates edited w/o a time frame would be a reasonable metric for that.

**Comment by MikeLud** [comment]
I am currently working with @algertc  on saving OCR training data so we can fine tune the OCR model. I already made changes to my ALPR module and @algertc is working on the changes to the ALPR-Dashboard.

Also I have over 5,000 image of all 50 states license plate that I will be using to fine tune the OCR model.

![image](https://github.com/user-attachments/assets/e3cf627e-cee8-473b-87fa-12e30ed6fc6c)

**Reply by deez2009** [reply]
Cool!  Does it matter if the OCR results and confidence is burned into the image?  Or should that be removed to make it cleaner?

**Reply by MikeLud** [reply]
Does it matter if the OCR results and confidence is burned into the image? **No, for the OCR training images**

Should that be removed to make it cleaner? **Only needed to be removed for the training images for the license-plate detection model.**

**Reply by deez2009** [reply]
Ok, thanks.  I haven't see much failure with license-plate detection at all.   OCR on CA plates struggles with "L/4", "0/O", "I/T" and the paper plates make it even worse, especially if we are capturing at less than ideal angles.  

I can't wait to see this feature.  :)

**Reply by MikeLud** [reply]
Below is a link to license plate image dataset there are more the 5,000 image there are 8,751 images

https://www.kaggle.com/datasets/gpiosenka/us-license-plates-image-classification

**Reply by deez2009** [reply]
Thanks Mike!
